pyproject.toml

```diff
@@ -14,7 +14,7 @@ classifiers=[
 [tool.poetry.dependencies]
 python=">=3.8.0,<3.12"
 numba=">=0.51.0"
-qibo = ">=0.1.13"
+qibo = { git = "https://github.com/qiboteam/qibo.git", branch = "fix_fused" }
```

Comment:
This can now be reverted to main or the latest release since qiboteam/qibo#950 is merged.

Reply:
Yes, I know. I think that @scarrazza would like to make a new release of qibo soon.
As soon as we release qibo 0.1.15 I will update the lock file.

Reply:
@andrea-pasquale the new release is now online, could please regenerate the lock file (and test)? Thanks.

Reply:
It seems that there are some issues. I will fix them.

Reply:
Tests are now passing locally on GPU for both qibo and qibojit (and they seems to pass also on GitHub). We should be ready to merge @scarrazza

Reply:
Thanks, feel free to merge when the CI completes. I will then make the new release.